azkaban-common/src/main/java/azkaban/user/Permission.java

```diff
@@ -191,6 +191,7 @@ public int toFlags() {
     SCHEDULE(0x0000008),
     METRICS(0x0000010),
     CREATEPROJECTS(0x40000000), // Only used for roles
+    UPLOADPROJECTS(0x0008000),
```

Comment:
You have discovered a critical bug and want to disallow uploads until resolution. As multiple people are in the admins list, you want to prevent accidental uploads.

If you are saying, keep it simple, and ADMIN's can upload and non-admins can't. We can make it that way. I largely implemented it the way CREATEPROJECTS is implemented in this project. There can be an ops group which has UPLOADPROJECTS permission, in which case admins need not pitch in.

Let me know your thoughts!